packages/iTwinUI-react/src/core/Table/Table.test.tsx

```diff
@@ -2727,6 +2727,156 @@ it('should render action column with column manager', () => {
   expect(actionColumn[3].textContent).toBe('View');
 });
 
+it('should render dropdown menu with specified class and style in action column with column manager', async () => {
```

Comment:
Should the new tests be made shorter? Or is the current length alright? I was thinking of making a common variable for the columns, or something similar, but I was not sure if that would be necessary.

Reply:
I think you don't need 3 new tests tbh. You can modify the existing column manager test to check for the inline style and iui-scroll, and you only need one new test for the dropdownMenuProps

Reply:
yeah, cleanup tests a little

Reply:
**[DONE]** Thanks, now we have only one new test and one edited existing test. I changed the existing test to test for the default style (`max-height: 315px`) and className of `iui-scroll`. The new test sees if the passed custom style is added and overrides the default style (`max-height`)